src/Cluster/RedisStrategy.php

```diff
@@ -55,4 +55,15 @@ public function getDistributor()
             'This cluster strategy does not provide an external distributor'
         );
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getDefaultCommands()
+    {
+        $commands = parent::getDefaultCommands();
+        $commands['WAIT'] = array($this, 'getKeyFromFirstArgument');
```

Comment:
Sorry I left for lunch with my comment nearly done but didn't notice your further inline comments before posting it :-) Anyway if we think about this in terms of which __node__ should be used instead of which __slot__, then I guess it would make more sense to let the decision to the actual implementation of the aggregate connection.